ironfish/src/network/peers/peerManager.ts

```diff
@@ -745,6 +742,7 @@ export class PeerManager {
     for (const peer of this.peers) {
       this.disconnect(peer, DisconnectingReason.ShuttingDown, 0)
     }
+    nodeDataChannel.cleanup()
```

Comment:
should this be on app teardown intead of here? If it isn't going to be globally torn down, should it at least by in PeerNetwork?

Reply:
I'll just delete this, I don't think we need it at this point